sirepo/pkcli/service.py

```diff
@@ -52,24 +59,67 @@ def flower():
         ])
 
 
+def flask():
+    from sirepo import server
+
+    use_reloader = pkconfig.channel_in('dev')
+    app = server.init(use_reloader=use_reloader)
+    # avoid WARNING: Do not use the development server in a production environment.
+    app.env = 'development'
+    app.run(
+        host=cfg.ip,
+        port=cfg.port,
+        threaded=True,
+        use_reloader=use_reloader,
+    )
+
+
 def http():
```

Comment:
We need a separate entry point for simple "http" flask so that we don't start he supervisor. It can be called anything because it is called from run-server.sh anyway.

Reply:
Are you looking for something different than the `def flask():` above?